src/components/profile/index.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import * as React from 'react'
+import type { BrowserHistory, HashHistory, MemoryHistory } from 'history'
+
+import ROUTE from 'constants/route'
+import InputField from 'components/common/input-field'
+
+import style from './index.style.js'
+
+export type RouterHistory = BrowserHistory | HashHistory | MemoryHistory
+
+export type ProfilePropsType = {
+  authError?: Object,
+  css?: Object,
+  children?: React.Node,
+  history?: RouterHistory,
+  email?: string,
+  password?: string,
+  signUp?: ({}) => boolean,
+}
+
+type StateType = {
+  email?: string,
+  password?: string,
+}
+
+export default class Profile extends React.Component<
+  ProfilePropsType,
+  StateType
+> {
+  state = {
+    email: 'test@test.com',
+    password: 'password',
+  }
+
+  render() {
+    const { authError } = this.props
+    return (
+      <div css={style.base}>
+        <p>{authError}</p>
+        {this.renderTitle()}
+        {this.renderForm()}
+      </div>
+    )
+  }
+
+  renderTitle = () => {
+    return (
+      <div css={style.title}>
+        <p css={style.title__text}>Profile</p>
+      </div>
+    )
+  }
+
+  renderForm = () => {
+    return (
+      <form onSubmit={this.handleSignUp}>
```

Comment:
This will not be a form since all fields are edited and submitted separately.

Check out how I used `onBlur` prop for input to save its value when the field loses focus.
Would be nice to have a reusable component that does that, and has an edit button built-in.

Reply:
`<form/>` is now `<Flex/>`